changelogs/current-change_log.md

```diff
@@ -21,9 +21,11 @@ Linker table for the many-to-many relationship between problem lists and hospita
 
 Table           | Attributes added | Attributes removed | 
 :-- |:-- |:--
-patient_condition     | added_date |
-patient_condition     | resolution_date |
-patient_condition     | is_deleted |
+patient_condition      | added_date |
+patient_condition      | resolution_date |
+patient_condition      | is_deleted |
+visit_observation_type | has_visit_observation |
+visit_observation_type | is_real_time |
```

Comment:
I think as well adding a summary or explanation of why the tables have changed would be useful.

Reply:
agreed

Reply:
done!
please feel free to push any clarifying changes. documentation is not my forté